wls-gui-wahllokalsystem/tests/types/vorfaelleundvorkommnisse/Ereignisart.spec.ts

```diff
@@ -46,4 +66,51 @@ describe("Ereignisart.ts", () => {
       expect(result).toStrictEqual(EreignisartEnum.Vorkommnis);
     });
   });
+
+  describe("getEreignisArtForDateRelatedToSchliessungsuhrzeitForBWB", () => {
+    beforeEach(() => {
+      const user = new User();
+      user.wahlbezirksArt = WahlbezirksArtEnum.BWB;
+      userStore.setUser(user);
+    });
+
+    it("should_returnVorkomniss_when_schliessungsuhrzeitIsNotSetAnd", () => {
```

Comment:
bitte überall den typo in `Vorkommnis` korrigieren
```suggestion
    it("should_returnVorkommnis_when_schliessungsuhrzeitIsNotSet", () => {
```